backend/src/main/java/ai/giskard/domain/ml/testing/TestExecution.java

```diff
@@ -16,6 +16,7 @@
 @NoArgsConstructor
 @Getter
 @Setter
+@Deprecated
```

Comment:
Should we simply remove it?

Reply:
For now we are still have the old test suite, I put has deprecated but I agree that the old test suite should be removed once the new one is fully implemented (including test suite creation from the UI)